lebedigital/demonstrator_scripts/computation_volume_content.py

```diff
@@ -0,0 +1,185 @@
+from lebedigital.unit_registry import ureg
+import pytest
+
+
+def computation_volume_content(input_dic):
+    """
+    This is the function to compute volume contents based on mass or volume ratios
+
+    The input and output is wrapped by the python pint package: https://pint.readthedocs.io/
+    This requires units to be attached to the input values.
+
+    Parameters
+    ----------
+    input_dic : dic / pint units required converted to matching units
+        required:
+            - 'density_cem'
+            - 'density_water'
+            - 'wb_mass_ratio' : water to binder ratio water/(cement + substitute (e.g. slag or cemII))
```

Comment:
cem II is not a substitute in itself, but if you substitute portland cement with slag or other binders, you get a cem II)

Reply:
I know and I know the wording is not 100% correct, but the computation is the same. 
I mean you can say you are substituting CemI for CemII.
All in all just semantics, the computation is the same.